src/modelldcatnotordf/modelelement.py

```diff
@@ -0,0 +1,84 @@
+"""Model element module for mapping a model element to rdf.
+
+This module contains methods for mapping a model element object to rdf
+for use in the modelldcat-ap-no specification._
+
+Refer to sub-class for typical usage examples.
+"""
+from typing import Optional
+
+from datacatalogtordf import URI
+from rdflib import Graph, Literal, Namespace, RDF, URIRef
+
+MODELLDCATNO = Namespace("https://data.norge.no/vocabulary/modelldcatno#")
+DCT = Namespace("http://purl.org/dc/terms/")
+DCAT = Namespace("http://www.w3.org/ns/dcat#")
+
+
+class ModelElement:
+    """A class representing a modelldcatno:ModelElement."""
+
+    __slots__ = ("_type", "_g", "_title", "_identifier")
+
+    _g: Graph
+    _title: dict
+    _identifier: URI
+
+    def __init__(self) -> None:
+        """Inits an object with default values."""
+        self._type = MODELLDCATNO.ModelElement
+
+    @property
+    def identifier(self) -> str:
+        """Get/set for identifier."""
+        return self._identifier
+
+    @identifier.setter
+    def identifier(self, identifier: str) -> None:
+        self._identifier = URI(identifier)
+
+    @property
+    def title(self) -> dict:
+        """Title attribute."""
+        return self._title
+
+    @title.setter
+    def title(self, title: dict) -> None:
+        self._title = title
+
+    def to_rdf(self, format: str = "turtle", encoding: Optional[str] = "utf-8") -> str:
+        """Maps the modelelement to rdf.
+
+        Args:
+            format: a valid format. Default: turtle
+            encoding: the encoding to serialize into
+
+        Returns:
+            a rdf serialization as a string according to format.
+        """
+        return self._to_graph().serialize(format=format, encoding=encoding)
+
+    def _to_graph(self) -> Graph:
+        """Returns the modelelement as graph.
+
+        Returns:
+            the modelelement graph
+        """
+        self._g = Graph()
+        self._g.bind("modelldcatno", MODELLDCATNO)
+
+        self._g.add((URIRef(self.identifier), RDF.type, URIRef(self._type)))
```

Comment:
Vi har ingen grunn til å tro at vi har en uri tilgjengelig, så ModelElement må legges til som BNode.

Eventuelt at den legges til som Bnode når identifier er None, og vanlig URIResource hvis den har identifier.

Reply:
Bra observert @NilsOveTen , da må vi nok tenke litt og ta en avgjørelse på det.

Reply:
Men da er det kanskje ikke noen annen mulighet enn blank node

Reply:
Eg har gjort det slik i datacatalogtordf:
```
    def _to_graph(self: Agent) -> Graph:

        if getattr(self, "identifier", None):
            _self = URIRef(self.identifier)
        else:
            _self = BNode()

        self._g.add((_self, RDF.type, FOAF.Agent))

        if getattr(self, "name", None):
            for key in self.name:
                self._g.add(
                    (
                        _self,
                        FOAF.name,
                        Literal(self.name[key], lang=key),
                    )
                )
```
https://github.com/Informasjonsforvaltning/datacatalogtordf/blob/b7fa28ecd43f6ddb22845b8a29d2a92ee3ad15b7/src/datacatalogtordf/agent.py#L111

Reply:
Hei dere, takk for våkne øyne og gode tips til dere begge! Da tror jeg vi gjør det som i eksempelet til @stigbd , for da er støtte for både dersom det finnes identifier og dersom ikke. Da blir det første mandag morgen å fikse dette! :-) Fortsatt god helg til dere! :-)

Reply:
Da tror jeg også vi kan gjøre tilsvarende med de andre klassene senere i modellen, denne situasjonen vil jo dukke opp igjen i forhold til de andre klassene også

Reply:
Da har jeg fikset en blank node slik som @stigbd gjorde, både på Modelelement og Property :-)